test/koji/run-build

```diff
@@ -36,8 +36,15 @@ def main():
     parser.add_argument('-v', '--verbose', action='store_true', help='Verbose output (unsupported)')
     opts = parser.parse_args()
 
+    # Splits 23 out of fedora-23
+    system = testinfra.DEFAULT_IMAGE.split("-")[-1]
+
     # Converts fedora-23 to f23
-    system = testinfra.DEFAULT_IMAGE[0] + testinfra.DEFAULT_IMAGE.split("-")[-1]
+    try:
+        number = int(system)
+        system = testinfra.DEFAULT_IMAGE[0] + system
+    except:
```

Comment:
Let's please avoid blanket except:s. These can hide all kinds of brokenness, even `SyntaxError` or `NameError`. Supposedly you want to guard the `int()` call here, so `except ValueError:`?